arangodb-net-standard.Test/DocumentApi/DocumentApiClientTest.cs

```diff
@@ -460,5 +464,42 @@ public async Task PutDocuments_ShouldNotThrowButReturnError_WhenDocumentIsNotFou
             Assert.True(updateResponse[0].Error);
             Assert.Equal(NOT_FOUND_NUM, updateResponse[0].ErrorNum);
         }
+
+        [Fact]
+        public async Task PatchDocumentsAsync_ShouldSucceed()
+        {
+            var postResponse = await _docClient.PostDocumentsAsync(_testCollection,
+               new[] {
+                    new { value = 1 },
+                    new { value = 2 },
+                    new { value = 3 }
+               });
+
+            var response = await _docClient.PatchDocumentsAsync(_testCollection,
+                new[] {
+                    new { postResponse[0]._key, value = 4 },
+                    new { postResponse[1]._key, value = 5 },
+                    new { postResponse[2]._key, value = 6 }
+                    });
+
+            Assert.Equal(HttpStatusCode.Accepted, response.Code);
+            Assert.Equal(3, response.Documents.Count);
+            Assert.NotNull(response.Documents[0]._key);
+            Assert.NotNull(response.Documents[0]._id);
+            Assert.NotNull(response.Documents[0]._rev);
+            Assert.NotEqual(postResponse[0]._rev, response.Documents[0]._rev);
+        }
+
+        [Fact]
+        public async Task PatchDocumentsAsync_ShouldThrow_WhenDocumentDoesNotExist()
```

Comment:
Method name is misleading - expectation is not 'ShouldThrow' but 'ShouldRecordError'